src/ex4/server/services/item_manager.js

```diff
@@ -21,17 +21,28 @@ class ItemManager {
     return regex.test(input);
   }
 
-  async addTask(taskInput, isCompleted) {
+  isRequestAdded(res) {
```

Comment:
what is this "res" that this method gets? it doesn't really explain it or what happens here.
If it's not an array you always return true, and if it's an array you return true only if it's not empty